CDP4Common/Poco/RelationalExpression.cs

```diff
@@ -41,7 +41,7 @@ public override string StringValue
                 return string.Format(
```

Comment:
perhaps this can be replaced with string interpolation

Reply:
Everything will then be shown in 1 line, which makes it much less readable.
We can make a multiline string interpolation, but the fact that there are white spaces between the to be inserted parameters also makes that less readable.
So, this is the best way for readability. 
The only risk is that this question is asked again in the future.
For that reason I would put a (Resharper?) comment here so we know this is done on purpose. 
@alexatrhea + @samatrhea: What's your opinion on this?

Reply:
I tend to agree with @lxatrhea here, that it's doing more good than bad and I would leave it as is.  Leave indeed a small comment above (doesnt need to be resharper).